modular_zubbers/code/game/machinery/smes.dm

```diff
@@ -0,0 +1,3 @@
+/obj/machinery/power/smes
```

Comment:
However, don't do it like that.

Instead, edit code\modules\power\smes.dm Line 67
```suggestion
/obj/machinery/power/smes/RefreshParts()
	SHOULD_CALL_PARENT(FALSE)
	var/power_coefficient = 1 // BUBBER EDIT CHANGE 
```

Reply:
👍